src/Pim/Bundle/CatalogBundle/Filter/ProductValueLocaleFilter.php

```diff
@@ -33,8 +34,30 @@ public function filterObject($productValue, $type, array $options = [])
     /**
      * {@inheritdoc}
      */
+    public function filterCollection($objects, $type, array $options = [])
```

Comment:
because filterCollection on parent reindex $objects with an other index, it broke completely the ProductValueCollection class which has `attribute_code-locale-scope` key.